test/cfa/ntp_conf_test.rb

```diff
@@ -131,88 +187,98 @@ def ntp_disk_file
 
   describe "#value" do
     it "obtains the value of the record" do
-      expect(record.value).to eq(augeas_element[:value].value)
+      record = ntp.records.first
+      expect(record.value).to eq("0.pool.ntp.org")
     end
   end
 
   describe "#value=" do
     it "sets the value of the record" do
-      value = "1.pool.ntp.org"
+      value = "3.pool.ntp.org"
+      record = ntp.records.first
       record.value = value
       expect(record.value).to eq(value)
+      ntp.save
+      expect(file.content).to include("server 3.pool.ntp.org iburst\n")
     end
   end
 
   describe "#comment" do
     context "when the record has not comment" do
       it "obtains nil" do
+        record = ntp.records.first
         expect(record.comment).to eq(nil)
       end
     end
 
     context "when the record has comment" do
-      let(:augeas_options) do
-        tree = CFA::AugeasTree.new
-        tree.add("#comment", "sample comment")
-        tree
-      end
-
       it "obtains the comment" do
-        expect(record.comment).to eq("sample comment")
+        record = ntp.records.to_a[1]
+        expect(record.comment).to eq("#sample comment")
       end
     end
   end
 
   describe "#comment=" do
     it "sets a comment to the record" do
-      comment = "sample comment"
+      comment = "#sample comment"
```

Comment:
Augeas saves what you pass as comment literally. I agree, it is necessary to indicate it.